src/tbots.py

```diff
@@ -0,0 +1,62 @@
+#!/opt/tbotspython/bin/python3.8
+
+import os
+import sys
+import iterfzf
+from subprocess import PIPE, run, check_call
+import argparse
+from thefuzz import fuzz
+from thefuzz import process
+
+THEFUZZ_MATCH_RATIO_THRESHOLD = 50
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser(description="Run stuff")
+
+    parser.add_argument("action", choices=["build", "run", "test"])
+    parser.add_argument("search_query")
+    parser.add_argument("-v", "--enable_visualizer", action="store_true")
```

Comment:
Can we add `-s` (open to suggestions) for `--stop_ai_on_start` aswell?

Reply:
added